bloom-instance/main.tf

```diff
@@ -35,4 +33,17 @@ locals {
     },
     local.default_tags
   )
-}
\ No newline at end of file
+}
+
+# Set up our network
+module "network" {
```

Comment:
I made a module because the codelab had one -- i think this is the right call but lmk if this is too much abstraction and I'll embed it within the top-level stuff.

Reply:
The module for the demo was more because I wanted to showcase how you would implement a module rather than because it was the best course of action for the networking components.  Treat a module the same way you would treat a function or class: use it when you want to duplicate the same set of resources with different values for the variables or when the templates get so complex that it makes sense to break them down into smaller, easier to grok chunks.  

In our case, the network components are a relatively small portion of the overall templates (compared to what the network portion usually looks like), so I think we're good to not put those in a module (YAGNI).

Reply:
Done, sg